### ISSUE TITLE
Documentation of performance and accessibility guide

### ISSUE THREAD

**Issue by Tlazypanda** [publiclab/plots2#8057]
**Please describe the desired behavior.**
Currently all the new changes introduced during Outreachy are not documented making it difficult for future contributors to maintain and improve on performance and accessibility.

Adding a performance and accessibility guide can help solve this problem by providing contributors links to changes made and offer tips on things to keep in mind while opening new pull requests.

**Comment by Tlazypanda** [comment]
Hey @jywarren as per our discussion on various issues that some sort of notes is required to keep track of changes maybe we can link this to the readme - https://hackmd.io/@itm2017004/BJuqtap6L initial draft :v:

**Comment by Tlazypanda** [comment]
Hey @jywarren @cesswairimu  I am almost done with this can you please have a look? Thanks :v:

**Comment by ebarry** [comment]
Yes, have it open now, will be reading during today! thank you @Tlazypanda for being part of the documentation revolution!

**Comment by ebarry** [comment]
Hi @Tlazypanda , it's so great to read documentation like this. Thank you!
Are any parts of this documentation aimed at non-coders, such as the people using the website directly? I was wondering this because of sections like https://hackmd.io/@itm2017004/BJuqtap6L#Avoiding-suspicious-link-text and the advice to avoid underlining.

**Comment by Tlazypanda** [comment]
Hey @ebarry I am not sure how links are rendered by the editor now but yes these can be used as general guidelines as well to make posts more accessible :sweat_smile: Originally this was meant for developers only but maybe we can extend it :v:

**Comment by cesswairimu** [comment]
Hey @Tlazypanda  how are you doing?  Did you want to move the documentation to the PL website or maybe make a markdown for it on the repo. Whats do you think? Thanks
